src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -33,6 +34,9 @@ const { feature, readOnly } = toRefs(props)
 
 const title = ref(feature.value.title)
 const description = ref(feature.value.description)
+const mediaLinkPopover = ref(null)
+const mediaImagePopover = ref(null)
+const mediaFilmPopover = ref(null)
```

Comment:
As you are only using these refs to close the popover (and it is the exact same action triggered on all refs) you could switch to a single ref name that is applied to all popover

```javascript
const mediaPopovers = ref(null) // will be an array of refs
// ...
function onAddMediaLink(description) {
    mediaPopovers?.forEach((popover) => popover.hidePopover())
    // ...
}
// ...
```
```html
<template>
    <!-- ... -->
    <DrawingStylePopoverButton ref="mediaPopovers" />
    <!-- ... -->
    <DrawingStylePopoverButton ref="mediaPopovers" />
    <!-- ... -->
</template>
```